app/services/code_climate/get_project_summary.rb

```diff
@@ -1,15 +1,31 @@
 module CodeClimate
   class GetProjectSummary < BaseService
-    def initialize(github_slug:)
-      @github_slug = github_slug
+    CODE_CLIMATE_API_ORG_NAME = 'rootstrap'.freeze
```

Comment:
Shouldn't this be an environment variable?

Reply:
You're totally right, it should!
As it was already like that before this PR, would you be fine if I changed it in another PR? I can probably start working on it right away